packages/compass-databases-navigation/src/collection-item.tsx

```diff
@@ -126,12 +141,15 @@ export const CollectionItem: React.FunctionComponent<
       isHovered={isHovered}
       isTabbable={isTabbable}
       onDefaultAction={onDefaultAction}
-      className={collectionItem}
+      className={cx(
+        collectionItem,
+        useNewSidebar ? collectionItemNewSpacing : collectionItemOldSpacing
```

Comment:
Something is off here, placeholders are rendered without enough padding on the left

![image](https://user-images.githubusercontent.com/5036933/182649082-d2dd7eef-b2db-4603-b787-44a2553c7dd0.png)

Reply:
Should be better now. Thanks for pointing it out.